internal/pkg/release/local_stored_collector.go

```
@@ -537,16 +537,34 @@ func (o *LocalStorageCollector) ReleaseImage(ctx context.Context) (string, error
-// getKubeVirtImage - CLID-179 : include coreos-bootable container image
-// if set it will be across the board for all releases
-func (o LocalStorageCollector) getKubeVirtImage(releaseArtifactsDir string) (v2alpha1.RelatedImage, error) {
+// kubeVirtArchName maps the Go/OCP architecture names used in the
+// ImageSetConfiguration to the keys used in the
+// 0000_50_installer_coreos-bootimages ConfigMap JSON (architectures.*).
+// Only architectures that actually ship a kubevirt container image are listed;
+// as of RHCOS 4.15+ those are x86_64 and s390x.
+// aarch64 and ppc64le are present in the bootimages JSON but carry no
+// kubevirt image entry, so they are intentionally omitted here.
+var kubeVirtArchName = map[string]string{
+	"amd64": "x86_64",
```

golangci-lint (2.12.2)

[error] string `amd64` has 10 occurrences, make it a constant

(goconst)